### PR TITLE
[FSLeyes plugin] changed Image.save/open with ads_utils.read/write

### LINKED ISSUE #257: Use adsutils for i/o in ADS plugin
Needs to be done after merging https://github.com/neuropoly/axondeepseg/pull/182

### PR DESCRIPTION
**Pull request by Stoyan-I-A**
fixes #257 and #299

### CODE REVIEW

**Comment by Stoyan-I-A** [comment]
The tooltip of the load_mask button has been updated. This fixes #299

File: ads_plugin.py [hunk 1]
```diff
@@ -86,10 +86,8 @@ def __init__(self, ortho, *args, **kwargs):
         load_mask_button.Bind(wx.EVT_BUTTON, self.on_load_mask_button)
         load_mask_button.SetToolTip(
             wx.ToolTip(
-                "Loads an existing axon or myelin mask into FSLeyes. "
-                "Please make sure the axon mask contains 'axon' in its name and "
-                "the myelin mask contains 'myelin' in its name. "
-                "The image must only contain 2 values: 0 for background and 255 for the region of interest."
+                "Loads an existing axonmyelin mask into FSLeyes. "
+                "The selected image should contain both the axon and myelin masks."
```
Comment (mariehbourget) [inline ads_plugin.py:90]: Are there specific values for background, myelin and axon? (ex: 0/128/255). If so, should we indicate them?

Comment (Stoyan-I-A) [reply ads_plugin.py]: done in 00e23d7

**Comment by mariehbourget** [comment]
@Stoyan-I-A is this PR ready to merge?

**Comment by Stoyan-I-A** [comment]
yes

**Pull request merged**